### ISSUE TITLE
Android: Add Android SDK Build-Tools 29.0.2 (Linux)

### ISSUE THREAD

**Issue by JesseScott** [microsoft/azure-pipelines-image-generation#1252]
**Tool information**
- Tool name: Android Build Tools
- Add or update? Update?
- Desired version: 29.0.2

**Areas and ecosystems**

- [x] Android
- [ ] C / C++
- [ ] Container tools
- [ ] Database servers or clients
- [ ] General purpose tools
- [ ] Go
- [ ] Java
- [ ] Javascript, Node.js, and TypeScript
- [ ] iOS, macOS, and Apple-specific tools
- [ ] .NET Core or .NET Desktop
- [ ] PHP
- [ ] Python
- [ ] Ruby
- [ ] Rust
- [ ] Scala
- [ ] Testing and code coverage
- [ ] Xamarin
- [ ] Other

**Image impacted**

ubuntu-latest

**Are you willing to submit a PR?**
Sure...

**Comment by JesseScott** [comment]
Possible yearly update to https://github.com/Microsoft/azure-pipelines-image-generation/issues/385

**Comment by JesseScott** [comment]
Currently getting the following on **vmImage** `ubuntu-latest`

```
Checking the license for package Android SDK Build-Tools 29.0.2 in /usr/local/lib/android/sdk/licenses
License for package Android SDK Build-Tools 29.0.2 accepted.
Preparing "Install Android SDK Build-Tools 29.0.2 (revision: 29.0.2)".
Warning: Failed to read or create install properties file.

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':app'.
> Failed to install the following SDK components:
      build-tools;29.0.2 Android SDK Build-Tools 29.0.2
  The SDK directory is not writable (/usr/local/lib/android/sdk)
```

**Comment by kaylangan** [comment]
Should be fixed in #1244. We'll roll out a new deployment with the fix.
